src/pages/business-adopters/index.tsx

```diff
@@ -0,0 +1,104 @@
+/**
+ * Copyright (c) Goodrain, Inc.
+ *
+ * This source code is licensed under the LGPL-3.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ **/
+
+/**
+ * Copyright (c) Goodrain, Inc.
+ *
+ * This source code is licensed under the LGPL-3.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import Layout from "@theme/Layout";
+import styles from "./styles.module.css";
+import clsx from "clsx";
+import React from "react";
+import Translate, { translate } from "@docusaurus/Translate";
+import { Typography } from "@douyinfe/semi-ui";
+
+export default function Partners() {
+  const { Text } = Typography;
+
+  const PartnersList = [
+    {
+      image: "/assets/images/huawei-501bd8d1790fd7f58a3a5af6c1b9107a.png",
```

Comment:
Does this image reference a picture from the repo?

Reply:
pic will be saved into this repository.